### PR TITLE
Travis implemented

### PR DESCRIPTION
**Pull request by blassoto**
### Summary

- Added travis integration
- Added badge status from travis

### Trello Card

https://trello.com/c/NvUsRJQh

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (franciscoabalan) [inline .rspec]: EOF

Comment (blassoto) [reply .rspec]: Done!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - bundle exec rails db:migrate
+script:
+  - bundle exec rspec spec -fd
+  - bundle exec rubocop app spec --format simple
+#notifications:
+#  email:
+#    recipients:
+#      - rails-bootstrap-dev@wolox.com.ar
+cache:
+  bundler: true
```
Comment (aleochoam) [inline .travis.yml]: EOF

File: .rspec (hunk 1 above)
Comment (aleochoam) [inline .rspec]: Rspec should not be part of this PR, we will get there in a second! But for this moment, can you please remove it?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - postgresql
+before_install:
+  - gem update --system
+before_script:
+  - cp config/database.travis.yml config/database.yml
+  - bundle exec rails db:create
+  - bundle exec rails db:migrate
+script:
+  - bundle exec rspec spec -fd
+  - bundle exec rubocop app spec --format simple
```
Comment (aleochoam) [inline .travis.yml]: And comment of remove this line until we get to rspec

File: config/database.travis.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+test: &test
+  adapter: postgresql
+  encoding: utf8
+  database: PJ-BackTraining-Blas_test
+  username: postgres
```
Comment (aleochoam) [inline config/database.travis.yml]: EOF

**Comment by blassoto** [comment]
Done!

File: Gemfile.lock [hunk 5]
```diff
@@ -148,7 +166,8 @@ DEPENDENCIES
   pg (~> 0.18)
   puma (~> 3.7)
   rails (~> 5.1.0)
-  rubocop
+  rspec-rails
```
Comment (franciscoabalan) [inline Gemfile.lock]: 👀

File: Gemfile [hunk 6]
```diff
@@ -38,5 +38,9 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :test do
+  gem 'rspec-rails'
```
Comment (joredudiaz) [inline Gemfile]: add version to lock the `rspec-rails` gem

File: Gemfile.lock (hunk 5 above)
Comment (blassoto) [reply Gemfile.lock]: Done!
I forgot to delete the gem, but now that is ok

File: Gemfile (hunk 6 above)
Comment (blassoto) [reply Gemfile]: When appropriate to implement testing I add it again with version

File: spec/spec_helper.rb [hunk 7]
```diff
@@ -0,0 +1,99 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```
Comment (aleochoam) [inline spec/spec_helper.rb]: Remove this file and we are good to go!

Comment (blassoto) [reply spec/spec_helper.rb]: Done!
